src/cmd/flux-job.c

```diff
@@ -337,7 +337,7 @@ static struct optparse_subcommand subcommands[] = {
     },
     { "urgency",
       "[OPTIONS] id urgency",
-      "Set job urgency",
+      "Set job urgency (0-31, HOLD, EXPEDITE, DEFAULT)",
```

Comment:
It's not clear from the usage message that the textual values are aliases for values in the 0-31 range.

Would it be more clear  to say
> Set job urgency (0-31), hold=0, default=16, expedite=31